airflow/providers/common/sql/operators/sql.py

```diff
@@ -198,7 +198,7 @@ class SQLExecuteQueryOperator(BaseSQLOperator):
     :param autocommit: (optional) if True, each command is automatically committed (default: False).
     :param parameters: (optional) the parameters to render the SQL query with.
     :param handler: (optional) the function that will be applied to the cursor (default: fetch_all_handler).
-    :param split_statements: (optional) if split single SQL string into statements (default: False).
+    :param split_statements: (optional) if split single SQL string into statements (default: see hook.run).
```

Comment:
What does *see hook.run* mean? Perhaps we should explain a bit more here.

Reply:
it's true it's true... was trying to stay on one line :)

Reply:
ok @uranusjr , updated